### ISSUE TITLE
Let the user pick the reference Store

### ISSUE THREAD

**Issue by r6d2** [alexschrod/barter-vg-enhancer#2]
Since trading is mostly done with people on another country, and regional pricing varies so much, I think it would be useful for each user to define which store is going to be used as a reference for pricing display and calculations.

So if I'm trading ROW keys/gifts I'd likely use EU or US stores, since they are consistent in most cases.

But occasionally picking the particular store of your fellow trader instead may allow for a better insight on how your offer will look to him.

_(Obvious note just in case: this is not the same as applying a currency conversion obtained from the web, you actually have to get the pricing for a particular store.)_

I'm not sure on how to implement such a thing, but from the script's standpoint this might be as simple as appending ?cc=eu -for instance- to the game URL, but it only seems to work if you're not logged in to Steam.

YMMV, though. Thanks for considering.

**Comment by ilyvion** [comment]
Yeah, the cc trick works, and your remark about it only working when you're not logged in is also correct. I'll see if I can't add it in. :)
